### ISSUE TITLE
AI eye cannot examine people blocked by structures, even if camera can see it.

### ISSUE THREAD

**Issue by Sindorman** [Aurorastation/Aurora.3#5390]
examine verb crashes if you examine someone who is blocked by a structure from AI eye

**Comment by Sindorman** [comment]
`Sorry, the following is not valid: ACE-v2\.5.0x300000a
usage: Examine mob|obj|turf
Sorry, the following is not valid: Examine
usage: Examine
usage: Examine is not accessible
`

**Comment by Sindorman** [comment]
Duplicate of #2576
